app/controllers/carts_controller.rb

```diff
@@ -18,19 +18,19 @@ def checkout_cart
     end
   end
 
-  def delete_item
+  def delete_project
     if params[:cart].nil?
-      delete_all_items_from_cart
+      delete_all_projects_from_cart
     else
-      delete_specific_item_from_cart
-      flash[:notice] = 'Item removed from cart'
+      delete_specific_project_from_cart
+      flash[:notice] = 'Project removed from cart'
     end
     redirect_to showcart_path
   end
 
-  def update_item_quantity
-    session[:cart][params[:update_item_quantity][:item_id]] = params[:update_item_quantity][:quantity]
-    flash[:notice] = 'Item quantity updated'
+  def update_project_quantity
+    session[:cart][params[:update_project_quantity][:project_id]] = params[:update_project_quantity][:quantity]
+    flash[:notice] = 'Project quantity updated'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.